public/static/pathways/her2_pathway.json

```diff
@@ -4,36 +4,43 @@
   "library": "mCODE_Library.cql",
   "criteria": [
     {
+      "id": "1",
```

Comment:
Do we need an id field for everything in an array? Is it not enough to just use indexes? That would make things simpler in a few places.

Reply:
No, because the id needs to be a way to always identify the object in the list. If we have 3 criteria and delete the middle one, then the identifier of the third criteria changes. I suppose we could make the front end remember the list ordering but I wanted to make the infrastructure versatile.